### DISCUSSION TITLE
[@types/chrome] missing chrome.instanceID

### DISCUSSION THREAD

**Discussion by WebGL3D** [DefinitelyTyped/DefinitelyTyped#65809] [Issues with a @types package]
[@types/chrome](https://www.npmjs.com/package/@types/chrome) contains type definitions for chrome extensions, but it appears to be missing the entire [chrome.instanceID](https://developer.chrome.com/docs/extensions/reference/instanceID) API.

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/11600745/129a7892-58df-4a30-a54f-c7b68d26bb20)

I'm using the current latest version of [this package](https://www.npmjs.com/package/@types/chrome) (`0.0.237`).

Additional context: [chrome.gcm](https://developer.chrome.com/docs/extensions/reference/gcm), a related, pairing API, is available as expected.

[a comment by an automated account is not shown]

**Comment by hxnir** [comment]
Hey all,
The related API - `gcm` will be deprecated in June 2024.
If possible I would love to help move this issue along, will opening a pull request with the relevant types help?

**Reply by nicolas377** [reply]
Hey! Could you send a link of the deprecation announcement? I can't seem to find anything related to `chrome.gcm` being deprecated.

**Comment by nicolas377** [comment]
Hey! Thanks for pointing this out, I'll look more into it and if needed, I'll open a PR ASAP.

**Reply by nicolas377** [reply]
Just an update, #67952 is up.

**Reply by hxnir** [reply]
Hey thanks a lot!
sorry for not updating sooner / opening the PR myself..

Here is a [link](https://bugs.chromium.org/p/chromium/issues/detail?id=831532) regarding the deprecation of the old `chrome.gcm.register` API.

**Comment by matcarlson** [comment]
Also feel free to send a PR if you'd like. We appreciate any help!
